packages/field-highlighter/theme/lumo/vaadin-field-outline-styles.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+import '@vaadin/vaadin-lumo-styles/spacing.js';
+import '@vaadin/vaadin-lumo-styles/style.js';
+import { css, registerStyles } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
+
+registerStyles(
+  'vaadin-field-outline',
+  css`
+    :host {
+      transition: opacity 0.3s;
+      -webkit-mask-image: none !important;
+      mask-image: none !important;
+    }
+
+    :host::before {
+      content: '';
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      box-shadow: 0 0 0 2px var(--_active-user-color);
+      border-radius: var(--lumo-border-radius);
+      transition: box-shadow 0.3s;
+    }
+
+    :host([context$='checkbox'])::before {
+      box-shadow: 0 0 0 2px var(--lumo-base-color), 0 0 0 4px var(--_active-user-color);
+    }
+
+    :host([context$='radio-button'])::before {
+      border-radius: 50%;
+      box-shadow: 0 0 0 3px var(--lumo-base-color), 0 0 0 5px var(--_active-user-color);
+    }
+
+    :host([context$='item'])::before {
+      box-shadow: inset 0 0 0 2px var(--_active-user-color);
+    }
+  `,
+  { moduleId: 'lumo-field-outline' }
+);
```

Comment:
The styles are no longer needed because we now set `overflow: visible` on the text-area. Registering styles for `vaadin-text-area` in field highlighter in undesirable because it would require certain import order.